registry/storage/vacuum.go

```diff
@@ -100,3 +100,18 @@ func (v Vacuum) RemoveRepository(repoName string) error {
 
 	return nil
 }
+
+// RemoveLayer removes a layer link path from the storage
+func (v Vacuum) RemoveLayer(repoName string, dgest digest.Digest) error {
```

Comment:
```suggestion
func (v Vacuum) RemoveLayer(repoName string, dgst digest.Digest) error {
```

nit: `dgst` is used everywhere else, just follow suite here.